.github/workflows/functional_tests.yml

```diff
@@ -27,11 +27,13 @@ jobs:
           PRIVATE_KEY: "${{ secrets.PRIVATE_KEY }}"
           VISITOR_ID: "${{ secrets.VISITOR_ID }}"
           REQUEST_ID: "${{ secrets.REQUEST_ID }}"
-      - name: Report Status
-        if: always()
-        uses: ravsamhq/notify-slack-action@0d9c6ff1de9903da88d24c0564f6e83cb28faca9
-        with:
-          status: ${{ job.status }}
-          notification_title: 'Python SDK Functional Test has {status_message}'
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  report-status:
+    needs: functional_tests
+    if: always()
+    uses: fingerprintjs/dx-team-toolkit/.github/workflows/report-workflow-status.yml@v1
+    with:
+      notification_title: 'Python SDK Functional Test has {status_message}'
```

Comment:
Will the {status_message} work here? Shouldn't it be `${{ jobs.functional_tests.status }}` ?

Reply:
It should, it is a [3rd-party action](https://github.com/ravsamhq/notify-slack-action) feature, sounds I should improve documentation for our reusable workflow